### PR TITLE
miatoll : New maintainer update

### PR DESCRIPTION
**Pull request by wileylau**

### CODE REVIEW

File: miatoll.json [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-			"paypal": "",
-			"telegram": "https://t.me/crdroid_miatoll"
-		}
-	]
-}
\ No newline at end of file
+  "response": [
+    {
+        "maintainer": "Jamie Ho (henloboi)",
+        "oem": "Xiaomi",
```
Comment (gwolf2u) [inline miatoll.json]: Leave Redmi as OEM

File: miatoll.json [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+        "device": "Redmi Note 9 Pro / 9 Pro Max / 9S / Poco M2 Pro",
```
Comment (gwolf2u) [inline miatoll.json]: Leave old device name (else breaks layout of device view from device)

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: miatoll.json (hunk 2 above)
Comment (wileylau) [reply miatoll.json]: Fixed.

File: miatoll.json (hunk 1 above)
Comment (wileylau) [reply miatoll.json]: Fixed.

File: miatoll.json [hunk 3]
```diff
@@ -2,8 +2,8 @@
   "response": [
     {
         "maintainer": "Jamie Ho (henloboi)",
-        "oem": "Xiaomi",
-        "device": "Redmi Note 9 Pro / 9 Pro Max / 9S / Poco M2 Pro",
+        "oem": "Redmi",
+        "device": "Redmi 9s series (miatoll)"",
```
Comment (gwolf2u) [inline miatoll.json]: Remove double quotes after (miatoll)"" as json doesn't validate

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Pull request merged**